### PR TITLE
[qa] Replace with openwisp-utils-qa-checks

### PR DESCRIPTION
**Pull request by waleko**
Replace old qa checks with `openwisp-utils-qa-checks` (related to openwisp/openwisp-utils#63 for this repo)

### CODE REVIEW

**Comment by waleko** [comment]
Travis is failing. Same problem as in openwisp/openwisp-network-topology#46

File: requirements-test.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
 coveralls
 responses
 freezegun
-openwisp-utils[qa]>=0.2.1
+openwisp-utils[qa]>=0.3.0
```
Comment (atb00ker) [inline requirements-test.txt]: Travis error related to the fact taht `openwisp-utils` is updated.
Please investigate and fix it! :smile:

Comment (nemesisdesign) [reply requirements-test.txt]: change `0.3.0` to `0.3.1`

File: requirements-test.txt [hunk 2]
```diff
@@ -1,7 +1,8 @@
 coveralls
 responses
 freezegun
-openwisp-utils[qa]>=0.2.1
+openwisp-utils[qa]>=0.3.1
+openwisp-users>=0.1.11
```
Comment (atb00ker) [inline requirements-test.txt]: openwisp-users is already in the `requirements.txt`, why have you added it here as well? :smile:

File: .travis.yml [hunk 3]
```diff
@@ -20,23 +20,11 @@ install:
   - pip install -r requirements-test.txt
 
 before_script:
-  - ./runflake8
-  - ./runisort
-  - ./runcheckmigration
-  # ensure there are no pending migrations
-  - ./tests/manage.py makemigrations openwisp_radius --dry-run | grep "No changes detected"
+  - openwisp-utils-qa-checks --migration-path "./openwisp_radius/migrations" --migration-module openwisp_radius
```
Comment (atb00ker) [inline .travis.yml]:
```
openwisp-utils-qa-checks --migration-path "./openwisp_radius/migrations" 
                         --migration-module openwisp_radius
```

**Pull request merged**